src/lib.rs

```diff
@@ -205,7 +206,7 @@ impl SandboxFS {
                 if !fs_attr.is_dir() {
                     warn!("Path {:?} is not a directory; got {:?}", &mappings[0].underlying_path,
                         &fs_attr);
-                    return Err(io::Error::from_raw_os_error(errno::Errno::EIO as i32));
+                    return Err(Error::from(io::Error::from_raw_os_error(errno::Errno::EIO as i32)));
```

Comment:
You might be able to use `.into()` here.

Reply:
I'm not sure about where you mean by "here"... I do not see an `into` in neither `io::Error` nor `failure::Error`) :-/

Also this line (and whole code block for that matter) will be gone with the upcoming mappings-related changes so I wouldn't worry too much. (Still interested in knowing what you meant though.)

Reply:
Where there's a from, there's an into: https://doc.rust-lang.org/std/convert/trait.Into.html#generic-implementations

Reply:
Aha, I see. Also found https://doc.rust-lang.org/rust-by-example/conversion/from_into.html before I saw your link. Changed here and also in `main.rs`.

However, I do not see guidance on when to use `from` vs. `into`...